client/extensions/woocommerce/app/settings/payments/payment-method-item.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ListItem from '../../../components/list/list-item';
+import ListItemField from '../../../components/list/list-item-field';
+
+const PaymentMethodItem = ( { method, translate } ) => {
+	return (
+		<ListItem>
+			<ListItemField>
+				{
+					method.isSuggested &&
+					(
+						<p className="payments__method-suggested">
+							{ translate( 'Suggested Method' ) }
+						</p>
+					)
+				}
+				<p className="payments__method-name">{ method.label }</p>
+			</ListItemField>
+			<ListItemField>
+				<p className="payments__method-information">{ method.fee }</p>
+				<p className="payments__method-information">
+					<a href={ method.information }>
+						{ translate( 'More Information' ) }
+					</a>
+				</p>
+			</ListItemField>
+			<ListItemField>
+				<Button compact>
+					{
+						translate( 'Set Up' )
```

Comment:
Minor: Reduce this translate call down to 1 line. Also looks like a good suggestion by the i18n bot.